docs/src/howto-guides/002_installation.md

```diff
@@ -164,6 +164,12 @@ Use uninstall script as shown below just to `remove` the `thin-edge.io` packages
 ```
 > Note: Removes just the thin-edge.io packages and does not remove the `configuration` files.
 
+The same thing can also be executed using a one-liner to download and run the script.
```

Comment:
I'd keep this one-liner command at the beginning of this uninstallation section itself. Let's keep the easiest thing first for the user. That's what we have done for the installation as well.

Reply:
The problem is that having it is not easy having the one-liner at the beginning as we still need two different instances of it (one for removing and one for purging). 

If we set the syntax, then the code snippet is not convenient for a copy/paste as they need to edit it before executing it.

```sh
curl -sSL https://raw.githubusercontent.com/thin-edge/thin-edge.io/main/uninstall-thin-edge_io.sh | sudo sh -s <remove|purge>
```

So I just added a note to the user that there are one-liners available in the beginning section. Commit ac66afa8a515a82bc39f4d0212eb38d027ddefc3

Reply:
So basically the uninstall script is a bit different to the install script, as the uninstall script has a required positional argument.